website/docs/r/federated_settings_org_config.html.markdown

```diff
@@ -38,12 +39,22 @@ data "mongodbatlas_federated_settings_org_configs" "org_configs_ds" {
 * `post_auth_role_grants` - (Optional) List that contains the default [roles](https://www.mongodb.com/docs/atlas/reference/user-roles/#std-label-organization-roles) granted to users who authenticate through the IdP in a connected organization.
 
 * `domain_restriction_enabled` - (Required) Flag that indicates whether domain restriction is enabled for the connected organization.
-* `identity_provider_id` - (Required) Unique 24-hexadecimal digit string that identifies the federated authentication configuration.
+* `identity_provider_id` - (Required) Legacy 20-hexadecimal digit string that identifies the UI access identity provider that this connected org config is associated with. This id can be found within the Federation Management Console > Identity Providers tab by clicking the info icon in the IdP ID row of a configured identity provider.
```

Comment:
Is this a fix in documentation or are we changing the format of id that is passed for this attribute?

Reply:
The documentation didn't match the [API Docs](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Federated-Authentication/operation/updateConnectedOrgConfig) and the experienced behavior. I updated the docs now, WDYT?

Reply:
sounds good, just saw @oarbusi comment that mentions that this can be obtained through `okta_idp_id` property (legacy format) in federated_identity_provider.